awspub/image.py

```diff
@@ -407,7 +407,9 @@ def create(self) -> Dict[str, _ImageInfo]:
                             },
                             "DeviceName": self.conf["root_device_name"],
                         },
+                        # TODO: make those ephemeral block device mappings configurable
```

Comment:
Should we make this a GH issue to not forget about it and give it more visibility?

Reply:
done (see https://github.com/canonical/awspub/issues/36)